hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cov_excl.el

```diff
@@ -0,0 +1,154 @@
+INSTANCE:prim_lfsr_tb.gen_duts[22].i_prim_lfsr
+CHECKSUM: "1437046094"
+INSTANCE:prim_lfsr_tb.gen_duts[23].i_prim_lfsr
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[10].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[11].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[12].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[13].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[14].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[15].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[16].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[17].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[18].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[19].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[20].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[21].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[22].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[23].i_prim_lfsr
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
```

Comment:
Please annotate ALL exclusions with the appropriate prefix, described in our methodology doc:
https://docs.opentitan.org/doc/ug/dv_methodology/#exclusions

Reply:
will do.